### DISCUSSION TITLE
How to use this module in win32 (32bit windows)?

### DISCUSSION THREAD

**Discussion by KasraCode** [letmaik/pyvirtualcam#75] [Q&A]
hello. I need to know how to use pyvirtualcam in windwos 32 bit and python 32 bit?

**Comment by letmaik** [comment]
32 bit is not supported. What's preventing you from upgrading to 64 bit?

**Reply by KasraCode** [reply]
I have to code a software to broadcast video into camera, for 32 bit computers and I can't use 64 bit because clients can't use.
Thanks for your reply.

**Reply by KasraCode** [reply]
do you have any suggestion for me to how can I create this software using python 32 bit?
